packages/graphql-generator/src/codegenSync/codegen.ts

```diff
@@ -0,0 +1,310 @@
+import {
```

Comment:
We should include a readme in the folder that points to where this code was taken from / include license information.